db/migrate/20211019122810_add_data_to_newbookings.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# adding data to paid and complete columns
+# rubocop:disable Metrics/MethodLength
+class AddDataToNewbookings < ActiveRecord::Migration[6.1]
+  def change
+    newbooking = Newbooking.all
```

Comment:
remember what we said about not using models (directly) in migrations - use special one-time model classes instead